app/src/main/res/values-de/strings.xml

```diff
@@ -48,7 +48,7 @@
     <string name="version">Version</string>
     <string name="sort_by">Sortieren nach</string>
     <string name="subscribe">Abonnieren</string>
-    <string name="unsubscribe">Abbestellen</string>
+    <string name="unsubscribe">Abonnement Kündigung</string>
```

Comment:
I would not agree to this change since it overcomplicates something that was already working and was already clear to the end user.